dev/run

```diff
@@ -6,11 +6,13 @@ AUTHZ_DB_DSN="postgres://postgres:xmtp@localhost:15432/postgres?sslmode=disable"
 NODE_KEY="8a30dcb604b0b53627a5adc054dbf434b446628d4bd1eccc681d223f0550ce67"
 
 go run cmd/xmtpd/main.go \
-    --message-db-connection-string "${MESSAGE_DB_DSN}" \
-    --message-db-reader-connection-string "${MESSAGE_DB_DSN}" \
-    --authz-db-connection-string "${AUTHZ_DB_DSN}" \
     --nodekey "${NODE_KEY}" \
     --metrics \
     --metrics-period 5s \
+    --store.enable \
```

Comment:
We'll also need an infra PR to update the command arguments to match these changes.